kuma/wiki/middleware.py

```diff
@@ -31,7 +31,8 @@ def process_request(self, request):
         remaps = DocumentZoneURLRemapsJob().get(request.locale)
         for original_path, new_path in remaps:
 
-            if request.path_info.startswith(original_path):
+            if (request.path_info == original_path or
+                request.path_info.startswith(''.join([original_path, '/']))):
```

Comment:
the original path could be a Unicode string, so this should be `u''.join([original_path, '/'])` or just `u'%s/' % original_path`